src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -199,10 +198,7 @@ page](https://developer.bluetooth.org/gatt/characteristics/Pages/CharacteristicV
 navigator.bluetooth.requestDevice({ filters: [{ services: ['heart_rate'] }] })
 .then(device => device.connectGATT())
 .then(server => server.getPrimaryService('heart_rate'))
-.then(service => {
-  if (!service) throw 'Heart Rate Service not found';
```

Comment:
Again - is it best practice to check for the service before returning? If not, no worries, happy to approve and merge.